src/plugins/event-plugins/__integ__/WebVitalsPlugin.test.ts

```diff
@@ -37,32 +37,35 @@ const removeUnwantedEvents = (json: any) => {
 test('WebVitalEvent records lcp and cls events', async (t: TestController) => {
     // If we click too soon, the client/event collector plugin will not be loaded and will not record the click.
     // This could be a symptom of an issue with RUM web client load speed, or prioritization of script execution.
-    await t.wait(300);
+    let browser = t.browser.name;
+    if (browser != 'Safari' && browser != 'Firefox') {
```

Comment:
nit: consider adding a return statement here to avoid nesting the test code below